docker-compose.yml

```diff
@@ -1,20 +1,18 @@
-version: "3.9"
-
-services:
-  db:
-    image: postgres
-    volumes:
-      - ./data/db:/var/lib/postgresql/data
-    env_file:
-      -  docker.env
-
-  web:
-    build: .
-    volumes:
-      - .:/code

```

Comment:
Let's leave it. It's necessary if you want to avoid building a new image for every code change